build/containers/deconstructsig/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM debian:buster-slim
+
+ENV LC_ALL=C.UTF-8
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        python3-pandas python3-click r-bioc-bsgenome r-cran-gplots procps && \
```

Comment:
python --> 3.7.3
python3-pandas --> 0.23.3
python3-click --> 7.0
r-bioc-bsgenome --> 1.50.0
r-cran-gplots --> 3.0.1.1
procps --> 2:3.3.15 (a bit weird how it is written)